draft-ietf-capport-api.md

```diff
@@ -52,15 +39,15 @@ This document describes an HTTP API that allows clients to interact with a Capti
 
 # Introduction
 
-This document describes a HyperText Transfer Protocol (HTTP) Application Program Interface (API) that allows clients to interact with a Captive Portal system. The API defined in this document has been designed to meet the requirements in the Captive Portal Architecture [I-D.ietf-capport-architecture]. Specifically, the API provides:
+This document describes a HyperText Transfer Protocol (HTTP) Application Program Interface (API) that allows clients to interact with a Captive Portal system. The API defined in this document has been designed to meet the requirements in the Captive Portal Architecture {{?I-D.ietf-capport-architecture}}. Specifically, the API provides:
```

Comment:
It gets better here:  You can use `{{?ARCH=I-D.ietf...}}` here and then `{{?ARCH}}` everywhere else.  The symbolic references are much nicer than opaque RFC references for most people.